examples/advanced/bionemo/task_fitting/src/bionemo_inference_processor.py

```diff
@@ -55,19 +43,8 @@ def create_task_data(self, task_name: str, fl_ctx: FLContext) -> Shareable:
         # get app root
         app_root = fl_ctx.get_prop(FLContextKey.APP_ROOT)
 
-        # Load model configuration to initialize training NeMo environment
-        self.base_config_path = os.path.join(app_root, self.base_config_path)
-        self.infer_config_path = os.path.join(app_root, self.infer_config_path)
-        base_config = OmegaConf.load(self.base_config_path)
-        infer_config = OmegaConf.load(self.infer_config_path)
-        config = OmegaConf.merge(base_config, infer_config)
-        self.log_info(fl_ctx, f"Load model configuration from {self.base_config_path} and {self.infer_config_path}")
-
-        # TODO send nemo checkpoint
-        configs = {BioNeMoConstants.CONFIG: OmegaConf.to_container(config)}
-
-        # convert omega conf to primitive dict
-        dxo = DXO(data=configs, data_kind=BioNeMoDataKind.CONFIG)
+        # task data is empty in this case
+        dxo = DXO(data={}, data_kind="EMPTY")
```

Comment:
why we don't need to send the configurations now?

Reply:
There are no configuration files anymore with the new bionemo inference command.